02-peer-protocol.md

```diff
@@ -1263,6 +1332,49 @@ Instead, the channel reserve is fixed at 1% of the total channel balance
 rounded down to the nearest whole satoshi or the `dust_limit_satoshis`,
 whichever is greater.
 
+If the opener requested a `option_will_fund` peer to contribute funds to
```

Comment:
`option_will_fund` doesn't exist, this doesn't need a feature bit since it instead relies on a TLV in `node_announcement`.